src/ButtonThread.h

```diff
@@ -159,9 +159,21 @@ class ButtonThread : public concurrency::OSThread
 
     static void userButtonDoublePressed()
     {
-#if defined(USE_EINK) && defined(PIN_EINK_EN)
+    #if defined(USE_EINK) && defined(PIN_EINK_EN)
         digitalWrite(PIN_EINK_EN, digitalRead(PIN_EINK_EN) == LOW);
-#endif
+    #endif
+    #if defined(GPS_POWER_TOGGLE)
+        if(config.position.gps_enabled)
+        {
+        DEBUG_MSG("Flag set to false for gps power\n");
+        } 
+        else 
+        {
+        DEBUG_MSG("Flag set to true to restore power\n");
+        }
```

Comment:
can you add a new string/branch at https://github.com/meshtastic/firmware/blob/de6b752db8874270cd1e82a7ed905e0361afc2ba/src/graphics/Screen.cpp#L533 and https://github.com/meshtastic/firmware/blob/de6b752db8874270cd1e82a7ed905e0361afc2ba/src/graphics/Screen.cpp#L471 to indicate that condition visually? 'disabled by button' or the like.